spark-3.5/clickhouse-spark/src/main/scala/xenon/clickhouse/ClickHouseCatalog.scala

```diff
@@ -238,6 +238,7 @@ class ClickHouseCatalog extends TableCatalog
       table: String,
       settingsClause: String
     ): Unit = {
+      log.debug(s"createTable($clusterOpt, $engineExpr, $database, $table, $settingsClause)")
```

Comment:
it will print the final executed SQL, I don't think this debug message would help.